src/tabs/tab-header-bar.scss

```diff
@@ -147,15 +147,16 @@ $label-horizontal-spacing: awsui.$space-xs;
 .tabs-tab:not(:last-child) {
   & > .tabs-tab-header-container {
     margin-inline-end: calc(-1 * #{awsui.$border-divider-section-width});
+    // This is the divider for the tab
     &:before {
       content: '';
       position: absolute;
       border-inline-end: awsui.$border-divider-section-width solid $separator-color;
-      inset: calc(#{awsui.$space-scaled-s}) 0;
+      inset: awsui.$space-scaled-s 0;
       opacity: 1;
     }
 
-    &.refresh:before {
+    &.refresh:not(.compact):before {
```

Comment:
We usually don't have to rely on a compact mode flag for these things. We have spacing tokens that should adjust their values based on compact/comfortable. Instead of using a compact flag, can we use a different spacing token that adjusts automatically?

For reference, here are the different spacing tokens: https://github.com/cloudscape-design/components/blob/main/style-dictionary/visual-refresh/spacing.ts#L68-L98